<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>res</class>
 <widget class="QDialog" name="res">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid white;
border-radius: 10px;
background: white;

background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgba(72, 145, 217, 255), stop: 1 rgba(255, 255, 255, 255));

</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_listres">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>370</y>
       <width>271</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Telecharger Liste Des Reservations</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_reservation">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>1031</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Entrer Numero De Billet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercherBillet">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CherBillet">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>420</y>
       <width>701</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton_triNumero">
        <property name="text">
         <string>Tri par NÂ°Billet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_triDestination">
        <property name="text">
         <string>Tri par Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_triHotel">
        <property name="text">
         <string>Tri par Hotel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_res_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>600</y>
       <width>271</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadB_Pack3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>530</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pack 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadB_Pack1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>490</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pack 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushStop">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>560</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius:8px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Icons/Sources/control_stop.png</normaloff>:/Icons/Sources/control_stop.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushPlay">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>560</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius:8px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Icons/Sources/control_play.png</normaloff>:/Icons/Sources/control_play.png</iconset>
     </property>
    </widget>
    <widget class="QVideoWidget" name="VideoInterface" native="true">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>483</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background: rgba(199, 153, 207, 100); 
border: rgba(134, 47, 147, 200);
}</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadB_Pack2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>510</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pack 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Mail</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>961</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::WrapLongRows</enum>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_32">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Smtp-server:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="server_3">
        <property name="text">
         <string>smtp.gmail.com</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_33">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Server port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="port_3">
        <property name="text">
         <string>465</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_34">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="uname_3">
        <property name="text">
         <string>aziz.bouzaiene1@gmail.com</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_35">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="paswd_3">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>22362125</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_36">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Recipant to:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="rcpt_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_37">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="subject_3"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_38">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Attachment:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_39">
        <property name="styleSheet">
         <string notr="true">background-color:none;</string>
        </property>
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLineEdit" name="file_3">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseBtn_4">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QPlainTextEdit" name="msg_3">
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="sendBtn_3">
     <property name="geometry">
      <rect>
       <x>766</x>
       <y>510</y>
       <width>117</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn_3">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>510</y>
       <width>116</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>651</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter Reservation</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NÂ°Billet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hotel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Numero_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Hotel_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Prix_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Destination_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox_ReservationA">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_A">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>651</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Modifier Avocats</string>
     </property>
     <widget class="QDialogButtonBox" name="buttonBox_ModifierReservation">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Prix_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hotel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Hotel_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NÂ°Billet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Destination_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Numero_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_M">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Entrer NÂ°Billet</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox_SupRes">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>81</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Billet_S">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>291</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
